drivers/i2c/muxes/i2c-mux-pca953xx.c

```diff
@@ -0,0 +1,1413 @@
+// SPDX-License-Identifier: GPL-2.0-only
```

Comment:
You've added this to drivers/i2c/muxes, but it's an I2C GPIO expander. It also appears to be a copy of /driver/gpio/gpio-pca953x, including duplicating a large number of the compatible strings.